plugins/module_utils/network/eos/argspec/ospfv3/ospfv3.py

```diff
@@ -127,19 +127,28 @@ def __init__(self, **kwargs):
                         "timers": {
                             "type": "dict",
                             "options": {
-                                "throttle": {
```

Comment:
I think I see what's happening here, but removing an option with no notice is not ideal. Is it possible to deprecate throttle, but retain the functionality by remapping its use onto the new commands?

Would probably also want `mutually_exclusive=[("throttle", "spf"), ("throttle", "lsa")]` if I understand correctly.

Reply:
@Qalthos Following is the error thrown by the appliance, when throttle is used

```
test(config-router-ospfv3)#timers throttle ?
  lsa  This command is deprecated by 'timers lsa tx delay initial'
  spf  This command is deprecated by 'timers spf delay initial'

```
I have retained the 'throttle' option, which when used, will throw the deprecation message. The user has to modify the playbook to use spf or lsa. 

I do not think we need the 'mutually_exclusive' option for spf and lsa. Correct me if I am wrong.